### PR TITLE
Add All channels link to dashboard template

### PR DESCRIPTION
**Pull request by peterp**
Please add a channels link in the left nav. This should be styled the same as the "All campaigns" link See screenshot:
screen shot 2013-07-17 at 4 36 57 pm

Please add the 'body' of the current channels page: http://qa-vumi.za.prk-host.net/channels/  to the 'body' of the dashboard page. This view is displayed when a user clicks "All channels" in the left nav. In other words, instead of having a separate channels page, we display the channels in the dashboard template when a user clicks "All channels" See screenshot: 
screen shot 2013-07-17 at 5 21 12 pm

### CODE REVIEW

File: go/conversation/templates/conversation/dashboard_base.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                        -->
+                        <li><a href="{% url 'conversations:index' %}?conversation_status=draft">Deactivated</a></li>
+                        <li><a href="{% url 'conversations:index' %}?conversation_status=finished">Archived</a></li>
+                    </ul>
+                </li>
+
+
+                <li>
+                    <a href="{% url 'channels:index' %}">All channels</a>
+                </li>
```
Comment (hodgestar) [inline go/conversation/templates/conversation/dashboard_base.html:55]: If this nav is shared perhaps it needs to be in a shared template?

Comment (peterp) [inline go/conversation/templates/conversation/dashboard_base.html:55]: That's why I placed it in `dashboard_base.html`

Comment (hodgestar) [inline go/conversation/templates/conversation/dashboard_base.html:55]: Ah. My bad. Doh. :)

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**